src/simplewallet/simplewallet.cpp

```diff
@@ -114,6 +114,8 @@ typedef cryptonote::simple_wallet sw;
 
 #define OUTPUT_EXPORT_FILE_MAGIC "Loki output export\003"
 
+#define OLD_AGE_WARN_THRESHOLD (30 * 86400 / DIFFICULTY_TARGET_V2) // 30 days
```

Comment:
This limit is going to hit all service node operators pretty much *always*, at which point it becomes an annoyance that everyone just ignores rather than anything useful.  It makes a bit more sense for Monero where you expect to only have a few inputs in your wallet, but it's going to be chaos for us.

I'd suggest we either drop the commit, or else put it behind an option that defaults to off.

Reply:
Reverted